source/changelog/r77.inc

```diff
@@ -0,0 +1,61 @@
+.. _changelog-r77:
+
+************************
+Release August 25th 2020
+************************
+Due to the summer holidays, the release has been some time coming (as announced previously).
+The following changes rolled out to production on August 25th 2020.
+
+API and Backend
+===============
+The following new features and improvements are now available in the API and the backend.
+
+* Added support for payment request failure handling (Payment Request Settings), allowing to configure how many times to retry payment requests, capture on card for instance. **(Preview)**
+
+    * By default only one attempt is made on each payment request, and not all payment providers are subject to retry handling.
+    * Feature must be configured to take effect.
+    
+* Added support for scheduling subscriber contact changes, both temporary and permanent.
+* Added support to cancel/stop deliveries without stopping the subscription.
+* New API for listing all generated Distribution Innovation transactions
+* Preliminary support for Additional Products
+* Added support for querying demand and reminder schedules, showcasing the planned generation time for Payment Demands and related Reminders.
+* Preiminary support for contracts in orders (binding the subscriber to a specific set of terms, subject to a fee if cancelled/changed ahead of time).
+* Fixed an issue that allowed ExternalInvoiceIdentifier configuration to be created multiple times.
+* Changed the behaviour of how Distribution Innovation Exports handles Address lines on the Subscriber Contact. Now extra lines are mapped to a separate information field.
+* Extended the Invoice Document model with additional text lines beyond the default Text description.
```

Comment:
I actually think this is not totally fixed. The API-part is not fixed until r79.

Reply:
Removed from bullet list.